lib/steps.js

```diff
@@ -793,8 +793,13 @@ When(/I answer randomly for at most ([0-9]+) (?:page(?:s)?|screen(?:s)?)/, { tim
   // Give each screenshots folder a time-stamped name in case the dev gets clever
   // and use the same Scenario to run multiple randomized tests.
   let timestamp = files.readable_date();
-  scope.paths.random_screenshots = `${ scope.paths.scenario }/random_answers_screenshots-${ timestamp }`;
+  let random_input_path = `${ scope.paths.scenario }/random_answers_screenshots-${ timestamp }`
+  scope.paths.random_screenshots = random_input_path;
   fs.mkdirSync( scope.paths.random_screenshots );
+  await scope.addToReport( scope, {
+    type: 'row',
+    value: 'Testing interview with random input'
```

Comment:
```suggestion
    value: `Testing interview with random input (in ${ random_input_path })`
```
just seems useful if someone wants to know where in the random tests you are